fares-data-build-tool/create-data
# feat: cfd 515 myfares UI tests

# Description

-   Adds UI tests for MyFares, makes LNUD the test user for UI tests, limits otherProducts to 100 in test, adds an error log for when a non-test user has >50 other products

# Testing instructions

-   Run UI tests
-  Ensure otherProducts are limited to 100 in test (I tested this locally)
-  Ensure an alert is triggered when a non-test user with more than 50 products loads the otherProducts page

# Type of change
